src/imitation_cli/utils/activation_function_class.py

```diff
@@ -0,0 +1,37 @@
+"""Classes for configuring activation functions."""
+import dataclasses
+from enum import Enum
+
+import torch
+from hydra.core.config_store import ConfigStore
+
+
+class ActivationFunctionClass(Enum):
+    """Enum of activation function classes."""
+
+    TanH = torch.nn.Tanh
+    ReLU = torch.nn.ReLU
+    LeakyReLU = torch.nn.LeakyReLU
+
+
+@dataclasses.dataclass
+class Config:
+    """Base class for activation function configs."""
+
+    activation_function_class: ActivationFunctionClass
+    _target_: str = "imitation_cli.utils.activation_function_class.Config.make"
```

Comment:
Is there a way to simplify this sort of thing so that it doesn't have to be repeated in every class?
(e.g. inheriting from a base class with an appropriate _target_ property)

Reply:
There probably is a super smart™ solution that is entirely incomprehensible to the casual user. This is one of the cases where I decided to accept some level of repetition for the sake of simpler understanding.